### PR TITLE
Fix errors in mongo.ts

### PR DESCRIPTION
**Pull request by EricJizbaMSFT**
message: 'Argument of type 'NoConnectionServer' is not assignable to parameter of type 'Server | PromiseLike<Server>'.
  Type 'NoConnectionServer' is not assignable to type 'PromiseLike<Server>'.
    Property 'then' is missing in type 'NoConnectionServer'.'
at: '130,9'

message: 'Type 'Promise<{}>' is not assignable to type 'Promise<string>'.
  Type '{}' is not assignable to type 'string'.'
at: '448,3'

### CODE REVIEW

**Comment by EricJizbaMSFT** [comment]
@chrisdias @sandy081 FYI here's my first PR for this repo. Let me know if you want me to assign you as reviewers going forward, otherwise I'll just have Jay review

**Comment by chrisdias** [comment]
Awesome! I added @sandy081 as a reviewer too (at least to start with) since he first authored the extension.

File: src/mongo/mongo.ts [hunk 1]
```diff
@@ -440,12 +434,9 @@ export class Collection implements IMongoResource {
...
 	}
 
-	private find(args?: any): Promise<string> {
-		const promise = new Promise((c, e) => {
-			let cursor = this.collection.find(args);
-			this.readNext([], cursor, 20, c);
-		});
-		return promise;
+	private find(args?: any): Thenable<string> {
+		return this.collection.find(args).limit(20)
```
Comment (whiskeyjay) [inline src/mongo/mongo.ts]: Magic number 20? A comment or define a constant would explain it better.

Comment (EricJizbaMSFT) [inline src/mongo/mongo.ts]: I was just maintaining the previous behavior. I changed it to be a const and filed an issue to improve this behavior: https://github.com/Microsoft/vscode-mongodb/issues/31

**Pull request merged**